index.html

```diff
@@ -253,27 +253,28 @@ <h3>
         <ul>
           <li>The <dfn id=
           'dom-PerformanceNavigationTiming-name'><code>name</code></dfn>
-          attribute MUST return the <code>DOMString</code> value of the
-          <a data-cite="!HTML/dom.html#the-document's-address">address</a> of
+          attribute MUST return the <code><dfn
+          data-cite="WEBIDL#es-DOMString">DOMString</dfn></code> value of the
+          <a data-cite="HTML/dom.html#the-document's-address">address</a> of
           the <a>current document</a>.
           </li>
           <li>The <dfn id=
           'dom-PerformanceNavigationTiming-entryType'><code>entryType</code></dfn>
-          attribute MUST return the <code>DOMString</code> "<code id=
+              attribute MUST return the <code><a>DOMString</a></code> "<code id=
```

Comment:
Sorry, I was not clear. You don't need `<code>` around `<a>` when the definition is wrapped in `code`. So:  

```suggestion
              attribute MUST return the <a>DOMString</a> "<code id=
```

The same applies to the other DOMString references.